medicines/doc-index-updater/src/document_manager/mod.rs

```diff
@@ -27,20 +33,36 @@ async fn del_document_handler(
         let duration = Duration::days(1);
 
         match queue.send(message, duration).await {
-            Ok(_) => Ok(warp::reply::json(
-                &state_manager.set_status(id, JobStatus::Accepted).await?,
-            )),
+            Ok(_) => Ok(state_manager.set_status(id, JobStatus::Accepted).await?),
             Err(_) => Err(warp::reject::custom(FailedToDispatchToQueue)),
         }
     } else {
         Err(warp::reject::custom(FailedToDispatchToQueue))
     }
 }
 
+async fn del_document_xml_handler(
```

Comment:
I'm assume `del` means `delete`?  shall we actually write `delete` or is there a decent reason to keep as `del`?

if changing to delete then the other areas below with del would need changing too but I won't pop a comment there

Reply:
agree: `delete` and `create` instead of `del` and `check_in` would align with the names for `*_manager` and `*Message` that we've got elsewhere